config/initializers/crossposter_config.rb

```diff
@@ -2,18 +2,18 @@
 
 Rails.application.configure do
   config.x.application_name = ENV.fetch("CROSSPOSTER_APP_NAME", "Mastodon Twitter Crossposter")
-  config.x.domain = ENV.fetch("CROSSPOSTER_DOMAIN")
+  config.x.domain = ENV.fetch("CROSSPOSTER_DOMAIN", nil)
```

Comment:
This is not an optional config. It is used at some points to determine if the post came from the crossposter, to inform the redirect url for the OAuth procedure, I think adding this as optional will break stuff